### DISCUSSION TITLE
Blocked by an object

### DISCUSSION THREAD

**Discussion by N0b0dy724** [Courseplay/Courseplay_FS22#3314] [Q&A]
Hello, 
I am trying to fix a tractor mod.  Currently when using Courseplay it will generate the course and start just fine but when it gets to a turn it stops and says blocked by an object.  I am trying to figure out which settings I need to adjust to see if I can get this working.  Any help would be greatly appreciated.

**Comment by Tensuko** [comment]
Screenshot with CP debug on and F5 active.

**Reply by N0b0dy724** [reply]
I appreciate the help!  I know it is a mod issue..just not sure what to change.

**Comment by N0b0dy724** [comment]
Let me know if you need something else

![fsScreen_2024_06_26_12_28_15](https://github.com/Courseplay/Courseplay_FS22/assets/137697965/aaa40645-4211-499b-a297-24b4a171aeb3)

**Reply by Tensuko** [reply]
I wrote what I need to see....

**Reply by N0b0dy724** [reply]
Did I get you what you needed?  Sorry not meaning to be a noob.

**Reply by Tensuko** [reply]
Now I can see the screenshot....
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/9af769fb-fdc7-4432-97cc-885a74fe2b1c)

You got to remove the old FS19 colision. That what looks like a strange arrow.

**Comment by FrankEliasPetro** [comment]
I have a problem with courseplay, when I go to plant, I create the route and at first it shows me a message that says "needs to be refilled", how can I solve it?

**Reply by Tensuko** [reply]
Can you put fertilizer in the machine? Is it in? If no turn off fertilizer in vehicle settings.
